MekHQ/src/mekhq/campaign/parts/CombatInformationCenter.java

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
This file's whitespace needs normalization.

Reply:
Fixed